.storybook/3.COLORS.stories.mdx

```diff
@@ -84,37 +85,35 @@ var(--color-primary-alternative)
 var(--color-primary-muted)
 var(--color-primary-inverse)
 
-var(--color-secondary-default) [DEPRECATED]
-var(--color-secondary-alternative) [DEPRECATED]
-var(--color-secondary-muted) [DEPRECATED]
-var(--color-secondary-inverse) [DEPRECATED]
-var(--color-secondary-disabled) [DEPRECATED]
-
 /** States */
 /** Error */
 var(--color-error-default)
 var(--color-error-default-hover)
 var(--color-error-default-pressed)
 var(--color-error-alternative)
+var(--color-error-default-hover)
+var(--color-error-default-pressed)
 var(--color-error-muted)
 var(--color-error-inverse)
 
 /** Warning */
 var(--color-warning-default)
+var(--color-warning-default-hover)
+var(--color-warning-default-pressed)
```

Comment:
These CSS variables are present and allowed to be used. Updating the docs to reflect this.

<img width="758" alt="Screenshot 2024-06-09 at 8 15 45 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/c5a70ea3-d996-49db-af52-dc6fbfab26aa">